app/templates/components/events/view/overview/event-sponsors.hbs

```diff
@@ -6,20 +6,15 @@
   <div class="header">{{t 'Event sponsors'}}</div>
 </div>
 <div class="content">
-  {{events/events-table
-    columns=columns
-    data=data
-    store=store
-    query=query
-    isNotStoreQuery = true
-    modelName = modelName
-    useNumericPagination=true
-    showGlobalFilter=true
-    showPageSize=true
-    moveToDetails='moveToDetails'
-    editEvent='editEvent'
-    deleteSponsor=deleteSponsor
-    editSponsor=editSponsor
-    customGlobalFilter='name'
-  }}
+  {{tables/default columns=columns
+               rows=model.data
```

Comment:
it should be data, not model.data @kushthedude

Reply:
still the same